cmake/Package.cmake

```diff
@@ -0,0 +1,23 @@
+# -- packaging -----------------------------------------------------------------
+
+set(CPACK_PACKAGE_VENDOR "Tenzir")
+set(CPACK_PACKAGE_CONTACT "engineering@tenzir.com")
+set(CPACK_VERBATIM_VARIABLES ON)
+
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
+set(CPACK_RESOURCE_FILE_README "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
+set(CPACK_INSTALLED_DIRECTORIES "/var/lib/vast" "/var/log/vast")
+
+set(CPACK_GENERATOR TZST;DEB)
+set(CPACK_DEBIAN_COMPRESSION_TYPE gzip)
+set(CPACK_DEBIAN_FILE_NAME DEB-DEFAULT)
+set(CPACK_DEBIAN_PACKAGE_NAME vast)
+set(CPACK_DEBIAN_PACKAGE_RELEASE 1)
+set(CPACK_DEBIAN_PACKAGE_SECTION "contrib/database")
+set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA
+    "${CMAKE_CURRENT_SOURCE_DIR}/debian/postinst"
+    "${CMAKE_CURRENT_SOURCE_DIR}/debian/postrm"
+    "${CMAKE_CURRENT_SOURCE_DIR}/debian/prerm")
+set(CPACK_DEBIAN_PACKAGE_DEBUG ON)
```

Comment:
Should this be removed before merging?

Reply:
I would keep it. The extra output isn't overly verbose:
```
CPackDeb:Debug: CPACK_TOPLEVEL_DIRECTORY          = '/home/tobim/t/vast/deb/build/gcc-static/redeb/_CPack_Packages/Linux/DEB'
CPackDeb:Debug: CPACK_TOPLEVEL_TAG                = 'Linux'
CPackDeb:Debug: CPACK_TEMPORARY_DIRECTORY         = '/home/tobim/t/vast/deb/build/gcc-static/redeb/_CPack_Packages/Linux/DEB/VAST-2.3.0-Linux'
CPackDeb:Debug: CPACK_OUTPUT_FILE_NAME            = 'vast_2.3.0-1_amd64.deb'
CPackDeb:Debug: CPACK_OUTPUT_FILE_PATH            = '/home/tobim/t/vast/deb/build/gcc-static/redeb/vast_2.3.0-1_amd64.deb'
CPackDeb:Debug: CPACK_PACKAGE_FILE_NAME           = 'VAST-2.3.0-Linux'
CPackDeb:Debug: CPACK_PACKAGE_INSTALL_DIRECTORY   = 'VAST 2.3.0'
CPackDeb:Debug: CPACK_TEMPORARY_PACKAGE_FILE_NAME = '/home/tobim/t/vast/deb/build/gcc-static/redeb/_CPack_Packages/Linux/DEB/vast_2.3.0-1_amd64.deb'
CPackDeb:Debug: CPACK_DEBIAN_PACKAGE_CONTROL_STRICT_PERMISSION = ''
CPackDeb:Debug: CPACK_DEBIAN_PACKAGE_SOURCE       = ''
```